python/vowpalwabbit/dftovw.py

```diff
@@ -634,6 +556,9 @@ def __init__(self, label_list):
 
         # Unpack columns
         columns = set()
+        # Override the type to Any as labels have complex machinery that is not yet communicated via its type.
+        label_list: List[Any] = label_list
```

Comment:
not sure why it is necessary ?

Reply:
This is needed because functionality exposed through AttributeDescriptors extensions are used just below and the ContextualbanditLabel and MultiLabel types do not expose it. So its a type error with those types as is.